test/check-machines-storage-pools

```diff
@@ -695,6 +695,59 @@ class TestMachinesStoragePools(VirtualMachinesCase):
             remove=True,
         ).execute()
 
+    def testGeneralCheckForPoolsDeletion(self):
+        b = self.browser
+        m = self.machine
+
+        args = self.createVm("subVmTest1")
+        m.execute("virsh vol-create-as images a 1M")
+        m.execute("virsh vol-create-as images z 1M")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual machines")
+
+        b.wait_in_text("#card-pf-storage-pools button .card-pf-title-link", "Storage pool")
+        b.click("#card-pf-storage-pools .pf-c-card__header button")
+        b.wait_visible("#storage-pools-listing")
+
+        # Check for deleting a pool whose disk is used by a VM
+        b.click("#pool-images-system-action-kebab button")
+        b.click("#pool-images-system-action-kebab a:contains(Delete)")
+        b.set_checked("#storage-pool-delete-volumes", True)
+        b.wait_in_text(".pf-c-backdrop .pf-c-helper-text__item-text", "Pool's volumes are used by")
+        b.wait_visible(".pf-c-backdrop footer button[aria-disabled=true]:contains(Delete)")
+
+        # Check "Cancel" button
+        b.click(".pf-c-backdrop footer button:contains(Cancel)")
+        b.wait_not_present(".pf-c-backdrop")
+
+        # Check "X" button of the dialog
+        b.click("#pool-images-system-action-kebab button")
+        b.click("#pool-images-system-action-kebab a:contains(Delete)")
+        b.click(".pf-c-backdrop button[aria-label=Close]")
+        b.wait_not_present(".pf-c-backdrop")
+
+        # Check that the error shown after the title, not the footer
```

Comment:
How about adding instead a pixel test for figuring out where the error is shown??

Reply:
If we add the pixel test you can also remove https://github.com/cockpit-project/cockpit-machines/pull/985/commits/0a482e73086e8705184f8143cd3c48493ac907ec as it's covered.

Reply:
OK, I add a pixel test in the new push, but it seems that I doesn't have the permission of cockpit-project/pixel-test-reference.git.